### ISSUE TITLE
Changing metrics in model.train()

### ISSUE THREAD

**Issue by santiactis** [divamgupta/image-segmentation-keras#243]
Hello there, first of all thank you for this amazing package.
I was wondering if it is possible to change the metric used during the model.train stage. In the source code I found that it is set up as 'accuracy' by default. 
I am new to this world so maybe I am missing something obvious. 
Thank you in advance!

**Comment by mavaylon1** [comment]
yes you can
